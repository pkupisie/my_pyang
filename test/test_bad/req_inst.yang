module req_inst {
  namespace urn:a;
  prefix a;

  typedef lref {
    type leafref {
      path "/foo";
    }
  }

  typedef iiref {
    type instance-identifier;
  }

  leaf foo {
    type empty;
    config false;
  }

  leaf bar {
    type lref {
      require-instance false; // error: not allowed in YANG 1
    }
  }

  leaf baz {
    type iiref {
      require-instance false;
    }
  }
}
