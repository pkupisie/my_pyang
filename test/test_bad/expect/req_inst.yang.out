req_inst.yang:22: error: BAD_RESTRICTION
